app/frontend/helpers/expectedErrors.js

```diff
@@ -0,0 +1,20 @@
+const expectedErrors = {
+  ConversationRatesError: () => '',
```

Comment:
a simple array would be sufficient to store these values if you don't intend to do something else

Reply:
Good point, I just reused the code from translations since I had plans to merge the two files, but later decided to solve that in later in #388 so I left it this way since it server the short-term purpose.